.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: rust:1.31
+      - image: rust:1.33
```

Comment:
why 1.33? do we need something that wasn't available in 1.31, maybe `pub(crate)`? otherwise why not 1.36?

Reply:
See the commit message.